index.js

```diff
@@ -16,7 +16,7 @@ const Action = require("./lib/action");
 const Asset = require("./lib/asset");
 const Prepare = require("./lib/prepare");
 const Rendition = require("./lib/rendition");
-const Storage = require("./lib/storage");
+const { Storage } = require("./lib/storage");
```

Comment:
If we do this here, we should do it for all the other imports too... (Prepare, Rendition, etc)